app/scripts/modules/core/src/pubsub/pubsubSubscription.service.ts

```diff
@@ -0,0 +1,15 @@
+import { module } from 'angular';
+
+import { API_SERVICE, Api } from 'core/api/api.service';
+
+export class PubsubSubscriptionService {
+  constructor(private API: Api) { 'ngInject'; }
+
+  public getPubsubSubscriptions(): ng.IPromise<string[]> {
```

Comment:
You should drop the `ng` namespacing by importing `IPromise` from `angular`.

Reply:
Done.